regression/cpp/Apple_extensions1/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
If there isn't already, could you raise an issue and add it as a comment at the bottom of test.desc ( @tautschnig )

Reply:
Done.